certbot-dns-google/certbot_dns_google/_internal/dns_google.py

```diff
@@ -306,7 +306,7 @@ def _find_managed_zone_id(self, domain: str) -> str:
 
             for zone in zones:
                 zone_id = zone['id']
-                if 'privateVisibilityConfig' not in zone:
+                if zone.get('privateVisibilityConfig') is not None or zone['visibility'] != "private": # pylint: disable=line-too-long
```

Comment:
Why not just forget about the check on `privateVisibilityConfig` ? This property only makes sense for a zone with its `visibility` set to `private`, and this is what we have to test here anyway.

Reply:
Make total sense @adferrand. Updated accordingly